pygmt/helpers/decorators.py

```diff
@@ -35,22 +35,18 @@
         frame : bool or str or list
             Set map boundary
             :doc:`frame and axes attributes </tutorials/basics/frames>`. """,
-    "timestamp": """\
+    "timestamp": r"""
         timestamp : bool or str
             Draw GMT time stamp logo on plot.""",
     "cmap": r"""
         cmap : str
            File name of a CPT file or a series of comma-separated colors
            (e.g., *color1*,\ *color2*,\ *color3*) to build a linear continuous
            CPT from those colors automatically.""",
-    "color": """\
-        color : str or 1-D array
-            Select color or pattern for filling of symbols or polygons [Default
-            is no fill].""",
-    "fill": """\
+    "fill": r"""
         fill : str
-            Select color or pattern for filling of symbols or polygons [Default
-            is no fill].""",
+            Set the color or pattern for filling the symbols or polygons
```

Comment:
```suggestion
            Set color or pattern for filling symbols or polygons
```
For me it reads much easier without the articles in the description, same for the following changes. What do other @GenericMappingTools/pygmt-maintainers think about that?

Reply:
I am fine with this suggestion :slightly_smiling_face:.

Reply:
Sounds good to me too.

Reply:
I removed the articels in the docstrings of the `fill` parameter in the commits https://github.com/GenericMappingTools/pygmt/pull/2351/commits/7cb5cf0affd63e96b1bfaa758885846de0fb2ff2 and https://github.com/GenericMappingTools/pygmt/pull/2345/commits/917aa4f0bd5b2b94ecc96a63c575c588dd935417.

Reply:
You know you can apply suggestions just by clicking the "Add suggestion to batch" button @yvonnefroehlich ?

Reply:
> You know you can apply suggestions just by clicking the "Add suggestion to batch" button @yvonnefroehlich ?

Yes, I know this option to combine code suggestions during review. But here only one occurrence was mentioned in the suggested changes above. Thus, I pushed a single "own" commit (https://github.com/GenericMappingTools/pygmt/commit/7cb5cf0affd63e96b1bfaa758885846de0fb2ff2) changing all occurrences at once (except `compressionfill` and `expressionfill` of `pygmt.Figure.meca`, which are updated via commit https://github.com/GenericMappingTools/pygmt/commit/917aa4f0bd5b2b94ecc96a63c575c588dd935417 in PR #2345).
Hm, but maybe I am missing your point here or/and you are right and I simply do not know something :sweat_smile:. In this case feel free to explain it again and I am happy to learn 🙂.

Reply:
No problem, just wanted to notice this point :wink: I meant the "Add suggestion to batch" button which allows to commit all accepted suggestions at once instead of individual actions.